charts/internal/calico/templates/node-cpva/configmap-node-vertical-autoscaling.yaml

```diff
@@ -0,0 +1,20 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-node-vertical-autoscaler
+  namespace: kube-system
+data:
+  node-autoscaler: |-
+    {
+      "calico-node": {
+        "requests": {
+          "cpu": {
+            "base": "120m",
+            "step": "80m",
+            "nodesPerStep": 10,
+            "max": "1000m"
```

Comment:
Dont know how vpa works internally, but does this mean calico-node can grow up to 1000m ? Or can this configmap be modified individually ?

Reply:
to my understanding its a normalization parameters used-only for calculating the resources and the limits not for capping the resources: 

```
//   If we find 64 cores and 4 nodes we get scalars of:
//     by-cores: 10 + (2 * (round(64, 4)/4)) = 10 + 32 = 42
//     by-nodes: 10 + (2 * (round(4, 2)/2)) = 10 + 4 = 14
//   The larger is by-cores, and it is less than Max, so the final value is 42.
```

Reply:
OK, kinda understood, will see.